gen_sign_cert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
This is just for testing, right? Or do we expect nodes to execute this?

Reply:
It's just temporary for testing. However, each node will run something like this.